src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -332,15 +332,25 @@ export class PanelComponent extends Disposable implements IThemable {
 			this.focusPreviousTab();
 			eventHandled = true;
 		}
-
 		if (eventHandled) {
 			event.preventDefault();
 			event.stopPropagation();
 		}
 	}
 
+	onTabHeaderFocusOut(e: Event): void {
+		if (!(<HTMLElement>e.currentTarget).contains((<any>e).relatedTarget)) {
+			this._tabHeaders.forEach(th => {
+				if (th.tab === this._selectedTab) {
```

Comment:
there could be 2 headers with tabIndex 0,right?

Reply:
Nope, as those tabIndexes get reset when change the focus.